### ISSUE TITLE
Cannot operate on graphs with gradients that are None.

### ISSUE THREAD

**Issue by drewlinsley** [JianGoForIt/YellowFin#9]
Error is thrown during the after_apply operation in the yellowfin class. My suggestion is to screen for Nones during the apply_gradients operation. This is similar to the apply_gradients operation in the official repo:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/optimizer.py#L426

**Comment by JianGoForIt** [comment]
Hi @drewlinsley, Thanks for the suggestion. We will incorporate it from the PR here soon.

https://github.com/JianGoForIt/YellowFin/pull/6.

Cheers,

Jian
